packages/cli/test/commands/init.test.ts

```diff
@@ -32,8 +33,9 @@ jest.mock('@salto-io/core', () => ({
       } as unknown as Workspace
     }
   ),
+  ensureWorkspaceDoesNotExist: jest.fn(),
 }))
-
+const mockEnsureWorkspaceDoesNotExist = jest.spyOn(core, 'ensureWorkspaceDoesNotExist')
```

Comment:
I think it's a better convention to call this:
```suggestion
const spyEnsureWorkspaceDoesNotExist = jest.spyOn(core, 'ensureWorkspaceDoesNotExist')
```
If you recall our conversation from yesterday spies and simple mocks behave differently. The variable name should hint to the type.

Reply:
Agreed.